hardhat.config.ts

```diff
@@ -10,15 +11,18 @@ dotenvConfig({ path: resolve(__dirname, "./.env") });
 
 chai.use(solidity);
 
-const { PRIVATE_KEY, ALCHEMY_KEY, HOSTNAME_L1, HOSTNAME_L2 } = process.env;
+const {
+  PRIVATE_KEY,
+  ALCHEMY_KEY,
+  HOSTNAME_L1,
+  HOSTNAME_L2,
+  ETHERSCAN_API_KEY,
+} = process.env;
 
-/* if (!PRIVATE_KEY) {
-  throw new Error("Please set your PRIVATE_KEY in your .env file");
-} */
+if (!PRIVATE_KEY || !ALCHEMY_KEY || !ETHERSCAN_API_KEY) {
```

Comment:
nice! it means we should add `ETHERSCAN_API_KEY` to github secrets, right?

Reply:
already added 👍🏻